### DISCUSSION TITLE
web: ios手机上播放pag文件首帧闪烁

### DISCUSSION THREAD

**Discussion by ligeisme** [Tencent/libpag#1824] [Q&A]
设计给了好几个pag文件只有这一个play的时候首帧闪烁，这个怎么排查呀，设计删掉第一帧也没用

**Comment by zhalice2011** [comment]
你好, 方便上传一下闪烁的 PAG 文件吗?

**Reply by ligeisme** [reply]
[breathe.pag.zip](https://github.com/Tencent/libpag/files/13247757/breathe.pag.zip)

**Reply by ligeisme** [reply]
哥，解决了，设计优化了下素材就好了，她说三维软件Blender直接生成png序列帧动画拖进ae里重新生成了一遍png序列帧动画 再生成就突然好了。。。
